src/main/java/spoon/generating/replace/ReplacementVisitor.java

```diff
@@ -32,18 +34,24 @@
  *
  * This class is generated automatically by the processor {@link spoon.generating.ReplacementVisitorGenerator}.
  */
-class ReplacementVisitor extends CtScanner {
+class ReplacementVisitor<E extends CtElement> extends CtScanner {
```

Comment:
A new formal type parameter added to element: 'E extends spoon.reflect.declaration.CtElement'.
Why do you need this?